tokenizers/src/cli.rs

```diff
@@ -55,33 +61,8 @@ fn shell(matches: &ArgMatches) -> Result<()> {
 }
 
 fn main() -> Result<()> {
-    let matches = App::new("tokenizers")
-        .version("0.0.1")
-        .author("Anthony M. <anthony@huggingface.co>")
-        .about("Generate custom Tokenizers or use existing ones")
-        .setting(AppSettings::SubcommandRequiredElseHelp)
-        .subcommand(
-            SubCommand::with_name("shell")
-                .about("Interactively test a tokenizer")
-                .arg(
-                    Arg::with_name("vocab")
-                        .long("vocab")
-                        .value_name("VOCAB_FILE")
-                        .help("Path to the vocab.json file")
```

Comment:
Aren't you losing the help prompt for the cli ?

Reply:
Well spotted, the rust doc fixes that now.